.github/workflows/publish-autoinstrumentation-e2e-images.yaml

```diff
@@ -24,12 +24,6 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Docker meta
-        id: meta
-        uses: docker/metadata-action@v5
-        with:
-          images: ghcr.io/open-telemetry/opentelemetry-operator/e2e-test-app-golang
```

Comment:
I would prefer keep using the `Docker meta` (to align with other files) and use the tag with branch name 
```
type=ref,event=branch
````

Reply:
Also I would avoid using `latest` tag, since we probably don't want to version these images.